src/server/mod.rs

```diff
@@ -132,7 +132,7 @@ where
             .map(|connection| connection.as_str())
             .unwrap_or("");
 
-        let connection_header_is_upgrade = connection_header_as_str.eq_ignore_ascii_case("upgrade");
+        let connection_header_is_upgrade = connection_header_as_str.split(',').any(|s| s.trim().eq_ignore_ascii_case("upgrade"));
```

Comment:
Hi @mu2019, awesome! I just put here the output of `fmt`

```suggestion
        let connection_header_is_upgrade = connection_header_as_str.split(',').any(|s| s.trim().eq_ignore_ascii_case("upgrade"));
        let connection_header_is_upgrade = connection_header_as_str
            .split(',')
            .any(|s| s.trim().eq_ignore_ascii_case("upgrade"));        
```

Thx!